gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1398,7 +1398,8 @@ private static void checkContentType(BlobInfo blobInfo) throws IllegalArgumentEx
   byte[] readAllBytes(BlobId blob, BlobSourceOption... options);
 
   /**
-   * Send a batch request.
+   * Send a batch request. If the {@code batchRequest} parameter contains more than 100 deletes the
```

Comment:
My rationale here was to be transparent with the user on the amount of traffic we generate (as it might mean $ :). I agree that this is very tied to the implementation but nevertheless this is how the API behaves now, if it changes we can drop the comment.

BTW I am not attached to this and suspected you would have asked me to drop it :) I'll remove this!